dali/pipeline/operators/displacement/displacement_filter_impl_cpu.h

```diff
@@ -110,6 +112,67 @@ class DisplacementFilter<CPUBackend, Displacement,
   USE_OPERATOR_MEMBERS();
 
  private:
+  // TODO(klecki) We could probably interpolate with something other than float,
+  // for example integer type
+  struct LinearCoefs {
+    LinearCoefs(float rx, float ry) : rx(rx), ry(ry), mrx(1.f - rx), mry(1.f - ry) {}\
+    LinearCoefs() = default;
+    float rx, ry, mrx, mry;
```

Comment:
To be compliant with GoogleStyle, there should be `rx_, ry_ etc...`

Reply:
No, as per GoogleStyle: _Data members of structs, both static and non-static, are named like ordinary nonmember variables. They do not have the trailing underscores that data members in classes have._